doc/data/messages/n/no-method-argument/good.py

```diff
@@ -1 +1,5 @@
-# This is a placeholder for correct code for this message.
+class Person:
+    greeting = "hello"
+
+    def print_greeting(self):
+        print(self.greeting)
```

Comment:
```suggestion
    def print_greeting(self):
        print("hello")
```
Can't we do that ? The self is unused but we're illustrating the no-method-argument message so it's not a problem, right ?

Reply:
Ah ok like this. Sure. I thought that would break the tests if more than the expected message was emitted.

Reply:
We disable everything except the message that is documented, this is why we can have faulty code for clarity ``isinstance(thing, Animal)`` when ``Animal`` is not defined for example).